releases/release-7.5.0.md

```diff
@@ -0,0 +1,271 @@
+---
+title: TiDB 7.5.0 Release Notes
+summary: 了解 TiDB 7.5.0 版本的新功能、兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 7.5.0 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：7.5.0
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v7.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.5.0#version-list)
+
+TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
+
+相比于前一个 LTS（即 7.1.0 版本），7.5.0 版本包含 [7.2.0-DMR](/releases/release-7.2.0.md)、[7.3.0-DMR](/releases/release-7.3.0.md) 和 [7.4.0-DMR](/releases/release-7.4.0.md) 中已发布的新功能、提升改进和错误修复，并引入了以下关键特性：
+
+## 功能详情
+
+### 可扩展性
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 性能
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 稳定性
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 高可用
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### SQL 功能
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 数据库管理
+
+* BR 支持备份和恢复表统计信息 [#issue号](链接) @[Leavrth](https://github.com/Leavrth) <!--**tw@hfxsd** 1437-->
+
+    BR 备份工具 从 v7.5.0 开始支持备份和恢复数据库统计信息，包括列、索引和表级别的统计信息。通过支持该功能，从备份中恢复的 TiDB 数据库不再需要手动运行统计信息收集任务或等待自动收集任务的完成，简化了数据库维护并提升高了恢复库的查询性能。可以通过 [`--ignore-stats=false`](/br/br-snapshot-manual.md#备份统计信息) 启用该功能。
+
+    更多信息，请参考[用户文档](/br/br-snapshot-manual.md#备份统计信息)。
+
+### 可观测性
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 安全
+
+* 功能标题 [#issue号](链接) @[贡献者 GitHub ID](链接)
+
+    功能描述（需要包含这个功能是什么、在什么场景下对用户有什么价值、怎么用）
+
+    更多信息，请参考[用户文档](链接)。
+
+### 数据迁移
+
+* Data Migration (DM) 支持拦截不兼容（破坏数据一致性）的 DDL 变更 [#9692](https://github.com/pingcap/tiflow/issues/9692) @[GMHDBJD](https://github.com/GMHDBJD) <!--**tw@hfxsd** 1523-->
+
+    在 v7.5.0 之前，使用 DM 的 Binlog Filter 功能只能迁移或过滤指定的 event，且颗粒度比较粗，例如只能过滤 ALTER 这种大颗粒度的 DDL Event，这种方式在某些业务场景会受限，如业务允许 Add Column，但是不允许 Drop Column，但之前的版本都会被 Alter 这个 Event 过滤。
+
+    因此，在 v7.5.0 在原先基础上细化了支持的 DDL event，如新增 Modify Column（修改列数据类型），Drop column 等会导致数据丢失、数据被截断、精度损失等问题的细粒度 DDL event，用户可以按需配置，同时还支持拦截这些 DDL，并报错提示用户，用户可以及时介入手工处理，避免对下游的业务数据产生影响。
+
+    更多信息，请参考[用户文档](https://github.com/pingcap/docs-cn/pull/15309)。
+
+* 支持实时更新增量数据校验的 checkpoint [issue号](链接) @[lichunzhu](https://github.com/lichunzhu) <!--**tw@ran-huang** 1496-->
```

Comment:
@lichunzhu Please review this note and provide the issue link and document link.